cdap-api/src/main/java/co/cask/cdap/api/workflow/WorkflowConfigurer.java

```diff
@@ -71,4 +71,10 @@
    * @param action to be added to the {@link Workflow}
    */
   void addAction(WorkflowAction action);
+
+  /**
+   * Adds a fork as a next sequential step in the {@link Workflow}
```

Comment:
fork is executed in parallel but its the next sequential node in the `Workflow`. may be will clean documentation a bit.